cstrike/addons/amxmodx/scripting/csgo_accounts.sma

```diff
@@ -86,10 +70,12 @@ public client_connect(id)
 
 	if (is_user_bot(id) || is_user_hltv(id) || !accountsEnabled) return;
 
-	get_user_authid(id, playerData[id][STEAM_ID], charsmax(playerData[][STEAM_ID]));
 	get_user_name(id, playerData[id][NAME], charsmax(playerData[][NAME]));
 
-	mysql_escape_string(playerData[id][NAME], playerData[id][SAFE_NAME], charsmax(playerData[][SAFE_NAME]));
```

Comment:
I see that changes from https://github.com/TheDoctor0/CSGOMod/commit/54351db1c5498d4487a47e0d298c14926968ee79 have been reverted in csgo_accounts.sma. You probably had older version on your branch. Definitely needs to be fixed.